jekyll/archived/snippets/language-guides-two-copy.md

```diff
@@ -0,0 +1,22 @@
+| Language Guide                          | Framework    | GitHub Repo Name
```

Comment:
I think as well as being added to the archived folder this one still needs deleting from `_includes/snippets`?